nvme-print.c

```diff
@@ -125,6 +132,16 @@ void nvme_show_predictable_latency_per_nvmset(
 	__u16 nvmset_id, const char *devname,
 	enum nvme_print_flags flags)
 {
+	struct print_ops *ops = nvme_print_ops(flags);
+
+	if (ops) {
```

Comment:
Seems the statement implementations are almost same so can be defined as a macro as below for example.
```c
#define nvme_print(name, flags, ...) \
    do { \
        struct print_ops *ops = nvme_print_ops(flags); \
        if (ops) { \
            if (ops->name) \
                ops->name(__VA_ARGS__); \
            return; \
    } while (0);
...
void nvme_show_predictable_latency_per_nvmset(
...
{
    nvme_print(predictable_latency_per_nvmset, flags, plpns_log, nvmset_id, devname);
```

Reply:
Sounds like a good idea.